lib/nostrum/gateway/message_delete_event.ex

```diff
@@ -0,0 +1,32 @@
+defmodule Nostrum.Gateway.MessageDeleteEvent do
```

Comment:
Do you think this is a proper path for this module? Do you think this shouldn't be under `lib/nostrum/struct/`?

Reply:
Not a proper place. I placed the module in that directory out of habit. Resolved in b4f4faf.